CSharpDriver.sln

```diff
@@ -39,8 +39,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MongoDB.Driver.Legacy.TestH
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MongoDB.Driver.Legacy.Tests", "tests\MongoDB.Driver.Legacy.Tests\MongoDB.Driver.Legacy.Tests.csproj", "{7A015030-6329-4FAD-B6E3-CED5ED53019C}"
 EndProject
-Project("{F184B08F-C81C-45F6-A57F-5ABD9991F28F}") = "MongoDB.Driver.Legacy.VB.Tests", "tests\MongoDB.Driver.Legacy.VB.Tests\MongoDB.Driver.Legacy.VB.Tests.vbproj", "{D197D685-5509-491E-A456-BA36C662CBB4}"
```

Comment:
Removed from the solution as it's only testing the legacy driver and was having issues with finding VB on Linux. Code is still there.

Reply:
Do we want to remove the VB.NET tests on Windows too? If we decide the VB.NET tests are no longer valuable, then we should remove the entire project rather than simply removing it from the solution.

Reply:
At the project level I couldn't find a nice way to only remove for Linux, so we discussed in a a standup and said this was a good temporary solution to get the ball moving forward.